backend/models/platform.py

```diff
@@ -21,13 +24,23 @@ class Platform(BaseModel):
     slug: Mapped[str] = mapped_column(String(length=50))
     fs_slug: Mapped[str] = mapped_column(String(length=50))
     name: Mapped[str] = mapped_column(String(length=400))
+    category: Mapped[str | None] = mapped_column(String(length=50), default="")
+    generation: Mapped[int | None]
+    family_name: Mapped[str | None] = mapped_column(String(length=1000), default="")
+    family_slug: Mapped[str | None] = mapped_column(String(length=1000), default="")
+    url: Mapped[str | None] = mapped_column(String(length=1000), default="")
+    url_logo: Mapped[str | None] = mapped_column(String(length=1000), default="")
     logo_path: Mapped[str | None] = mapped_column(String(length=1000), default="")
 
     roms: Mapped[list[Rom]] = relationship(back_populates="platform")
     firmware: Mapped[list[Firmware]] = relationship(
         lazy="selectin", back_populates="platform"
     )
 
+    aspect_ratio: Mapped[float] = mapped_column(
```

Comment:
I think having the `aspect_ratio` as a small char column will be simpler to maintain in the long term, to avoid issues with floating point precision. We could always handle it as `<width_units>:<height_units>` for simplicity (e.g. `16:9`), and if for some reason we need its decimal representation, parsing and calculating it would be very simple.

Reply:
Okay sounds reasonable

Reply:
Done, changed to char and converting in the frontend properly